README.md

```diff
@@ -1,4 +1,34 @@
-# managed-velero-operator
+# Managed Velero Operator
+
+## Summary
+
+The Managed Velero Operator is used for backups in OpenShift Dedicated v4. It is based on an open source project [Velero]( https://velero.io) that uses a controller model where it monitors custom resources and takes actions based on their states. The Managed Velero Operator  dynamically creates and configures prerequisites for Velero and it then deploys the Velero software on the OpenShift Dedicated cluster. Velero backs up the Kubernetes object store: all the deployments, pods, config maps, secrets, etc. In addition to Kubernetes configuration pieces Velero also handles backing up and any persistent volumes that are attached to the cluster, for OpenShift Dedicated v4 these are AWS EBS volumes.
+
+## What the Managed Velero Operator Does
+
+1. When the Managed Velero Operator starts, it checks whether it is installed on a supported platform. For OpenShift Dedicated v4 environment it validates that 
+	+ it is installed on AWS
+	+ it has been installed with installer provisioned infrastructure
+	+ it has all the needed details in the cluster's infrastructure configuration to provision Velero
+
+2. Next, the Managed Velero Operator begins tge **Reconcile loop**. It checks whether the Velero custom resources are created/installed and it ensures that they are created/installed before it takes any further action.
+
+3. Next, the Managed Velero Operator starts up the manager and controller and waits for the initial configuration.
+
+4. Once the Managed Velero Operator detects its custom resources and understands that it's ready to provision Velero in this cluster it will check if an S3 bucket already have been provisioned to store the Kubernetes object store, the metadata and the details of the backup.  If the Managed Velero Operator detects that there isn't an S3 bucket defined in its custom resources, it will provision an S3 bucket for that use. **The Managed Velero Operator will ensure the following settings when the bucket is created and it will check periodically to ensure that the bucket settings are re-enforced.**
```

Comment:
```suggestion
4. Once the Managed Velero Operator detects its custom resources and understands that it's ready to provision Velero in this cluster, it will check if an S3 bucket already has been provisioned to store the Kubernetes object store, the metadata and the details of the backup. If the Managed Velero Operator detects that there isn't an S3 bucket defined in its custom resources, it will provision an S3 bucket for that use. **The Managed Velero Operator will ensure the following settings when the bucket is created and it will check periodically to ensure that the bucket settings are re-enforced.**
```

Removes a double space, adds punctuation for clarity and changes `have been provisioned` to `has been provisioned`.